packages/cli/src/commands/apps/create.ts

```diff
@@ -15,7 +15,14 @@ export default class AppCreateCommand extends InputOutputAPICommand<AppRequest,
 		...InputOutputAPICommand.flags,
 		authorize: flags.boolean({
 			description: 'authorize Lambda functions to be called by SmartThings',
-		})}
+		}),
+		principal: flags.string({
+			description: 'use this principal instead of the default when authorizing lambda functions',
+		}),
+		'statement-id': flags.string({
+			description: 'use this statement id instead of the default when authorizing lambda functions',
+		}),
```

Comment:
Since these flags are used so often, It might be worth putting these in some common place and `...`ing them in so if we need to update the text, we can just do it in one place.

Reply:
Good idea. I moved them